model/config.go

```diff
@@ -597,10 +599,38 @@ func (o *Config) IsValid() *AppError {
 	return nil
 }
 
-func (me *Config) GetSanitizeOptions() map[string]bool {
+func (o *Config) GetSanitizeOptions() map[string]bool {
 	options := map[string]bool{}
-	options["fullname"] = me.PrivacySettings.ShowFullName
-	options["email"] = me.PrivacySettings.ShowEmailAddress
+	options["fullname"] = o.PrivacySettings.ShowFullName
+	options["email"] = o.PrivacySettings.ShowEmailAddress
 
 	return options
 }
+
+func (o *Config) Sanitize() {
+	if len(*o.LdapSettings.BindPassword) > 0 {
```

Comment:
We want the field to be blank in the System Console when no password has been set.